docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -344,15 +344,16 @@ Exchange spooling is responsible for storing and managing spooled data for
 fault-tolerant execution. You can configure a filesystem-based exchange manager
 that stores spooled data in a specified location, such as :ref:`AWS S3
 <fte-exchange-aws-s3>` and S3-compatible systems, :ref:`Azure Blob Storage
-<fte-exchange-azure-blob>`, or :ref:`Google Cloud Storage <fte-exchange-gcs>`.
+<fte-exchange-azure-blob>`, :ref:`Google Cloud Storage <fte-exchange-gcs>`,
+or :ref:`HDFS <fte-exchange-hdfs>`.
 
 Configuration
 ^^^^^^^^^^^^^
 
 To configure an exchange manager, create a new
 ``etc/exchange-manager.properties`` configuration file on the coordinator and
 all worker nodes. In this file, set the ``exchange-manager.name`` configuration
-propertry to ``filesystem``, and additional configuration properties as needed
+property to ``filesystem`` or ``hdfs``, and additional configuration properties as needed
```

Comment:
```suggestion
property to ``filesystem`` or ``hdfs``, and set additional configuration properties as needed
```
nit, to make it grammatically correct & clear

Reply:
Done